docs/content/how-to/kubevirt/troubleshooting-kubevirt-cluster.md

```diff
@@ -0,0 +1,63 @@
+# Troubleshooting a KubeVirt cluster
+
+## Fetching VM bootstrap logs
+
+The VM console logs are really useful to troubleshoot issues when the HyperShift/KubeVirt VM nodes are not correctly joining the cluster.
+KubeVirt v1.1.0 will expose logs from the serial console of guest VMs in a k8s native way (`kubectl logs -n <namespace> <vmi_pod> -c guest-console-log`) but his is still not available with KubeVirt v1.0.0.
+
+On KubeVirt v1.0.0 you can use a helper script to stream them from interactive console executed in background.
+
+```bash
+#!/bin/bash
+NAME=$1
+
+if [[ -z "${NAME}" ]]
+then
+    echo "Please specify the name of the guest cluster."
+    exit 1
+fi
+
+HSNS=clusters
```

Comment:
we aren't guaranteed the namespace will be `clusters`. That's that the namespace typically is, but it doesn't have to be. 

maybe we should require a namespace as well for the hosted cluster

Reply:
OK, let's use an env variable with a reasonable default to keep the list of parameters short.